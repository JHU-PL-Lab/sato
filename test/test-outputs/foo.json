[
  {
    "err_input_seq": [ 0 ],
    "err_location": [
      "Clause",
      [ "Var", [ "Ident", "sum" ], null ],
      [
        "Binary_operation_body",
        [ "Var", [ "Ident", "x" ], null ], "+", [ "Var", [ "Ident", "t" ], null ]
      ]
    ],
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ [ "Ident", "t" ] ],
          "err_match_val": [ "Value_body", [ "Value_bool", true ] ],
          "err_match_expected": [ "Int_type" ],
          "err_match_actual": [ "Bool_type" ]
        }
      ]
    ] 
  }
]
