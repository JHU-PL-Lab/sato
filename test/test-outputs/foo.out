Type errors for:
- Input sequence  : [0]
- Found at clause : sum = x + t
--------------------
* Value    : t = true
* Expected : int
* Actual   : bool
Found in 261 steps
1 errors found. No further control flows exist.
